bots/pr/src/main/java/org/openjdk/skara/bots/pr/CommitCommandWorkItem.java

```diff
@@ -110,11 +111,12 @@ private void processCommand(Path scratchPath, HostedCommit commit, LimitedCensus
                 printer.println("` can only be used in pull requests.");
             }
         } else {
-            printer.print("Unknown command `");
-            printer.print(command.name());
-            printer.println("` - for a list of valid commands use `/help`.");
+            printer.printf(MessageFormat.format("""
+                    I see that you entered a comment with `/{0}` at the beginning of a line.
+                    `/{0}` is not a recognized Skara PR command, so it was ignored and treated like an ordinary comment.
+                    If you intended to issue a Skara command, use `/help` in a comment to see a complete list of Skara commands.
+                    """, command.name()));
```

Comment:
Since we are keeping the mail filtering rules the same as command execution, I don't think we should change this message.

Reply:
Sure, I reverted this change.